sdk/core/azure-core-amqp/src/impl/rust_amqp/rust_amqp/rust_wrapper/src/lib.rs

```diff
@@ -14,7 +14,7 @@ pub mod amqp;
 pub mod call_context;
 pub mod model;
 pub mod runtime_context;
-pub mod rust_error;
+//pub mod rust_error;
```

Comment:
maybe remove this line ?

Reply:
I have profound hopes that I'll be able to bring back the rust_error type. But yeah, it make ssense to remove it.